### ISSUE TITLE
disease

### ISSUE THREAD

**Issue by anngvu** [nf-osi/nf-metadata-dictionary#33]
This concept is currently being used out in the wild but is not found in the schema. It seems sufficiently useful, and no term of same meaning already exists in schema. 

Recommendation: ?

**Comment by allaway** [comment]
Could this be captured under/merged with `Diagnosis`?

**Comment by anngvu** [comment]
Yes, definitely seems redundant with `diagnosis`. Here are the fileview tables where `disease` appears. Where this column isn't even being used, e.g. [syn15666830](https://www.synapse.org/#!Synapse:syn15666830/tables/), we can definitely remove it. In [syn7850572](https://www.synapse.org/#!Synapse:syn7850572/tables/) and syn7817226, `disease` _is_ being used, but alongside `diagnosis` and largely duplicating information in the `diagnosis` column, so there we can remove `disease` too. [syn15666923](https://www.synapse.org/#!Synapse:syn15666923/tables/) is the only slightly more interesting one where `diagnosis` is _not_ being used.  The value [Orphanet:638](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=185) can be easily translated to "Neurofibromatosis type 1" and carried over into `diagnosis`. For `NF1 - MPNST`, that row will have "Neurofibromatosis type 1" in `diagnosis` and `tumorType` = "Malignant Peripheral Nerve Sheath Tumor".  We don't need to worry about translating `tumor` since `tumorType` is filled (though values need to be updated, as there are lots of abbreviations like "MPNST"). This will allow getting rid of `disease` -- at least in the table, not IRL, alas.

|fileview    |unique_values_for_term                                                                                                     |
|:-----------|:-------------------------------------------------------------------------------------------------------------------|
|syn15666830 |                                                                                                                    |
|syn15666651 |NA                                                                                                                  |
|syn7850572  |Control                    , Neurofibromatosis 1        , Not Applicable             , Control,Neurofibromatosis 1, |
|syn15666926 |                                                                                                                    |
|syn15666932 |                                                                                                                    |
|syn15667121 |                                                                                                                    |
|syn15666931 |                                                                                                                    |
|syn7817226  |Neurofibromatosis 1        , Control                    , Not Applicable             , Control,Neurofibromatosis 1, |
|syn15666933 |NA                                                                                                                  |
|syn15666930 |NA                                                                                                                  |
|syn15666928 |                                                                                                                    |
|syn15666677 |NA                                                                                                                  |
|syn15666927 |                                                                                                                    |
|syn15666937 |                                                                                                                    |
|syn15666923 |, Orphanet:638, tumor       , NF1 - MPNST                                                                           |

**Comment by allaway** [comment]
>at least in the table, not IRL, alas.

😅 if only it was that easy!
